draft-ietf-httpbis-client-hints.md

```diff
@@ -93,9 +93,9 @@ This document defines a new response header, Accept-CH, that allows an origin se
 
 Client Hints mitigate the performance concerns by assuring that clients will only send the request headers when they're actually going to be used, and the privacy concerns of passive fingerprinting by requiring explicit opt-in and disclosure of required headers by the server through the use of the Accept-CH response header.
 
-This document also defines an initial set of Client Hints.
+This document defined the Client Hints infrastructure, but does not define any specific features that will use it. Those features will be defined in their respective specifications.
```

Comment:
s/defined/defines? Past tense doesn't make sense here, I think?

Is "infrastructure" well understood term, or should we elaborate a bit on what this means? Something like, a framework for advertising hints that the client wants to receive, persisting them, and corresponding server logic? Ish?

Reply:
> s/defined/defines? Past tense doesn't make sense here, I think?

Oops. definitely "defines".

I'll also elaborate on "infrastructure"